### DISCUSSION TITLE
Android app icon switcher and widgets

### DISCUSSION THREAD

**Discussion by GinoRamirexNubox** [community/community#4146] [Mobile]
Hello everyone, a few weeks ago I changed from iphone to an android phone and I missed you the ios widgets and also being able to change the application icon appealing to the user experience to be the same, regardless of the device / operating system that  i use

**Comment by aconsuegra** [comment]
Thanks for your requests! We've added them to our internal tracking issue, but I can't promise when we'll have time to work on them. Please, keep the feedback coming!
